src/Mod/PartDesign/fcgear/svggear.py

```diff
@@ -57,14 +52,64 @@ def curve(self, *points):
     def close(self):
         self.svg.append('Z')
 
+
 if __name__ == '__main__':
     from optparse import OptionParser
-    p = OptionParser()
-    p.add_option('-a', '--angle', help='pressure angle',
-                 dest='angle', default=20)
+    p = OptionParser(
+        usage="usage: %prog [options] [[MODULE] NUMER_OF_TEETH]",
+        description=("Generates the outline of a metric, involtue gear. "
```

Comment:
involtue->involute

Reply:
thx, fixed. This has even slippet through our CI's linter.